demo/src/app/components/datepicker/datepicker.module.ts

```diff
@@ -106,6 +108,12 @@ const DEMOS = {
     type: NgbdDatepickerConfig,
     code: require('!!raw-loader!./demos/config/datepicker-config'),
     markup: require('!!raw-loader!./demos/config/datepicker-config.html')
+  },
+  positiontarget: {
```

Comment:
Could you please place this **before** the "config" demo for consistency → "config" is always last demo for all components.